### PR TITLE
use GNUInstallDirs for detecting install paths

### PR DESCRIPTION
**Pull request by kraj**
This helps with multilib builds

Signed-off-by: Khem Raj <raj.khem@gmail.com>

### CODE REVIEW

File: sources/ade/CMakeLists.txt [hunk 1]
```diff
@@ -47,12 +47,14 @@ if(BUILD_ADE_DOCUMENTATION)
...
-        ARCHIVE DESTINATION lib
-        LIBRARY DESTINATION lib
-        RUNTIME DESTINATION lib
-        INCLUDES DESTINATION include)
+	ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR}
+        LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR}
+        RUNTIME DESTINATION ${CMAKE_INSTALL_LIBDIR}
+	INCLUDES DESTINATION ${CMAKE_INSTALL_INCLUDEDIR})
 
 install(EXPORT adeTargets DESTINATION share/ade COMPONENT dev)
```
Comment (mshabunin) [inline sources/ade/CMakeLists.txt:59]: Shouldn't the following install commands be changed too? E.g. `share` -> `${CMAKE_INSTALL_DATAROOTDIR}`.

Comment (mshabunin) [reply sources/ade/CMakeLists.txt:59]: Also, please keep the file formating consistent.

**Comment by kraj** [comment]
Sure will send v2

File: sources/ade/CMakeLists.txt (hunk 1 above)
Comment (dmatveev) [reply sources/ade/CMakeLists.txt:59]: Done

**Review by dmatveev — APPROVED** [review]
Thanks @kraj, but I had to push some fixes to your "master" branch directly as the PR was opened from it. Hope it is ok

**Pull request merged**